EventsExpress.Core/Services/UsersService.cs

```diff
@@ -249,10 +251,20 @@ public IEnumerable<UserDTO> GetUsersByCategories(IEnumerable<CategoryDTO> catego
                     .Any(category => categoryIds.Contains(category.Category.Id)))
                 .Distinct()
                 .AsEnumerable();
-
+            
```

Comment:
Please remove these redundant spaces. Consider fixing any warnings that you have in this PR.

Reply:
removed